docs/tutorials/create-a-course.md

```diff
@@ -11,6 +11,44 @@ Specific requirements for this tutorial:
  - Python (>= 3.8) installed on your machine.
  - A POSIX System (e.g., Linux, BSD, Mac, WSL (Windows)).
 
+## Getting Started
+
+Before diving into the tutorial, ensure you meet the prerequisites and understand the recommended setup.
+This section provides guidance on preparing your environment.
+
+### Prerequisites
+
+1. Python (>= 3.8):
+
+   If Python is not already installed on your system,
+   follow the [Python Beginner’s Guide to Downloads](https://www.python.org/about/gettingstarted/) to install the appropriate version for your operating system.
+
+2. Docker:
+
+   To use Docker with the autograder, ensure Docker is installed and properly set up on your system.
+   Follow the [Docker Get Started Guide](https://docs.docker.com/get-started/) for installation and basic usage instructions.
+
+3. POSIX-Compliant System:
+
+   This tutorial assumes a POSIX system (Linux, BSD, macOS, or WSL for Windows) is being used.
+   Windows users should set up WSL for compatibility.
```

Comment:
I don't think any of this should be here.
This all belongs in the general README (which it already does).
We can point to it, but we don't want to have the same info in too many places.